deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/JobPropertiesParser.java

```diff
@@ -59,13 +59,18 @@
     private static final String EXCLUDE_KEYS_IN_EXPORT_NAME = "excludeKeysInExport";
     @Expose(serialize = true, deserialize = true)
     @SerializedName(EXCLUDE_KEYS_IN_EXPORT_NAME)
-    private boolean excludeKeysInExport;
+    private Boolean excludeKeysInExport;
 
```

Comment:
Why not use primitive?

Reply:
validating the get value as NULL or not in new changes. Not possible in boolean.

Reply:
Why would excludeKeysInExport be null? what is it set in swagger?

Reply:
Ideally we shouldn't validate anything on the client side. Why are we validating this?

Reply:
Are you sure we shouldn't be validating? I see our Import Export APIs (without storage Auth) validating mandatory parameters. Unit tests are also making sure we do this.

If that's not required, I can remove this change.

Reply:
Yeah we were doing that in the very beginning time of SDK. However we got away from that practice  because we can keep track of service changes. So we don't validate anymore and reply on service to validate for us.